profiles/Abhishek-00.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/101923614/213878832-4282e481-ce0c-4a57-a3e6-6f3eeb688901.png",
+    "issueId": 1670,
+    "name": "Abhishek Kumar",
+    "username": "Abhishek-00"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```